### PR TITLE
[pack] Consume Ready to Run binaries from extension bundle

### LINKED ISSUE #5630: Consume binaries from extension bundle with ReadyToRun enabled
To significantly improve Cold Start all extensions need to  built with ReadyToRun enabled.
  <PublishReadyToRun>true</PublishReadyToRun>

We will need to build packages for win-x86, win-x64 and linux-x64 and at runtime load the appropriate  bundles based on the bitness of the process.

### PR DESCRIPTION
**Pull request by soninaren**
resolves #5630

### CODE REVIEW

File: src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs [hunk 1]
```diff
@@ -241,5 +261,43 @@ private static string FindBestVersionMatch(VersionRange versionRange, IEnumerabl
 
             return bundleVersions.OrderByDescending(version => version.Version).FirstOrDefault()?.ToString();
         }
+
+        public async Task<string> GetExtensionBundleBinPath()
```
Comment (brettsam) [inline src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs]: nit -- end in `Async`

**Review by brettsam — COMMENTED** [review]
Have you been able to test all of these in prod to make sure they behave correctly?

**Review by safihamid — DISMISSED** [review]
:shipit:

File: src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs (hunk 1 above)
Comment (soninaren) [reply src/WebJobs.Script/ExtensionBundle/ExtensionBundleManager.cs]: Fixed the method name.

**Comment by soninaren** [comment]
> Have you been able to test all of these in prod to make sure they behave correctly?

Tested locally on windows and linux, have not tested in prod. Going to use artifacts from the PR to verify prod and merge.

**Review by safihamid — APPROVED** [review]
:shipit:

**Comment by soninaren** [comment]
@brettsam tested bundles in prod. Works as expected. we should be good to merge this.

**Pull request merged**